templates/stamp/stamp.json

```diff
@@ -0,0 +1,652 @@
+            "serverIpPrefix": "10.0.0.",
+            "mongoServerIpOffset": 10,
+            "mysqlServerIpOffset": 15,
+            "jbNetworkSecurityGroupName": "[concat(parameters('clusterName'),'-jb-', 'nsg')]",
+            "vmssNetworkSecurityGroupName": "[concat(parameters('clusterName'), '-vmss-', 'nsg')]",
+            "mongoNetworkSecurityGroupName": "[concat(parameters('clusterName'), '-mongo-', 'nsg')]",
+            "mysqlNetworkSecurityGroupName": "[concat(parameters('clusterName'), '-mysql-', 'nsg')]",
+            "clusterJumpBoxName": "[concat(parameters('clusterName'), '-jb')]",
+            "clusterJumpBoxDomainName": "[concat(parameters('clusterName'), 'jb')]",
+            "clusterMongoDataNodeName": "[concat(parameters('clusterName'), '-mongo')]",
+            "clusterMongoDataNodeDomainName": "[concat(parameters('clusterName'), 'mongo')]",
+            "clusterMysqlDataNodeName": "[concat(parameters('clusterName'), '-mysql')]",
+            "clusterMysqlDataNodeDomainName": "[concat(parameters('clusterName'), 'mysql')]",
+            "clusterVirtualMachineScaleSetName": "[concat(parameters('clusterName'), '-vmss')]",
+            "clusterVirtualMachineScaleSetDomainName": "[concat(parameters('clusterName'), 'vmss')]",
+            "jumpBoxAvailabilitySetFaultDomainCount": "1",
+            "jumpBoxNodeCount": "1",
+            "mongoDataNodesAvailabilitySetFaultDomainCount": "3",
+            "mongoDataNodeCount": 3,
+            "mysqlDataNodesAvailabilitySetFaultDomainCount": "3",
+            "mysqlDataNodeCount": 3,
+            "loadbalancerSettings": {
+                "loadBalancerName": "[variables('loadBalancerName')]",
+                "loadBalancerDomainName": "[concat(parameters('clusterNameDomainName'), '-', parameters('deploymentSlot'))]",
+                "natPoolName": "[concat(parameters('clusterName'), '-natpool')]",
+                "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontEnd')]",
+                "lbPoolID": "[concat(variables('lbID'),'/backendAddressPools/', variables('bePoolName'))]",
+                "bePoolName": "[variables('bePoolName')]",
+                "lbProbeLMSID": "[concat(variables('lbID'),'/probes/tcpProbeLMS')]",
+                "lbProbeCMSID": "[concat(variables('lbID'),'/probes/tcpProbeCMS')]",
+                "natStartPort": 50000,
+                "natEndPort": 50119,
+                "natBackendPort": 22,
+                "frontendNodeCount": "[parameters('frontendNodeCount')]",
+                "vmssAutoscaleSettingNameSuffix": "[variables('vmssAutoscaleSettingNameSuffix')]"
+            }
+        },
+        "ubuntuOsConfigs": {
+            "osName": "ubuntu",
+            "installerBaseUrl": "http://repo.mongodb.org/apt/ubuntu",
+            "installerPackages": "mongodb-org",
+            "imagePublisher": "Canonical",
+            "imageOffer": "UbuntuServer",
+            "imageSku": "16.04.0-LTS"
```

Comment:
Hopefully mongo/mysql can stay on Ubuntu16, but we'll need to downgrade app vmss to Ubuntu14 or Ubuntu12 to support edx installation.